orcoursetrion/tests/test_cmd.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Test command line parsing and actions
+"""
+import sys
+
+import mock
+
+from orcoursetrion.cmd import execute
+from orcoursetrion.tests.base import TestGithubBase
+
+
+class TestGithubCommand(TestGithubBase):
+    """Verify github commands via command line"""
+
+    def test_cmd_create_export_repo(self):
+        """
+        Command line test of create_export_repo
+        """
+
+        sys.argv = [
+            'orcoursetrion', 'create_export_repo',
+            '-c', self.TEST_COURSE,
+            '-t', self.TEST_TERM,
+            '-d', self.TEST_DESCRIPTION,
+        ]
```

Comment:
Shouldn't sys.argv be mocked instead for better isolation of future tests?

E.g.
```python
args = [
    'orcoursetrion', 'create_export_repo',
    '-c', self.TEST_COURSE,
    '-t', self.TEST_TERM,
    '-d', self.TEST_DESCRIPTION,
]

with mock.patch(sys, 'argv', args):
    with mock.patch('orcoursetrion.cmd.actions') as mocked_actions:
        execute()
        # ...
```